config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -272,9 +272,9 @@ presets:
   - name: LOG_DUMP_SYSTEMD_SERVICES
     value: containerd containerd-installation
   - name: KUBE_MASTER_EXTRA_METADATA
-    value: user-data=/workspace/github.com/containerd/cri/test/e2e/master.yaml,containerd-configure-sh=/workspace/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/workspace/test-infra/jobs/e2e_node/containerd/cri-master/env
+    value: user-data=/go/src/github.com/containerd/cri/test/e2e/master.yaml,containerd-configure-sh=/go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env=/go/src/test-infra/jobs/e2e_node/containerd/cri-master/env
```

Comment:
Shouldn't this be /go/src/k8.sio/test-infra?

Reply:
Actually, it should be /workspace/test-infra as it was